go/tests/glide_test.go

```diff
@@ -0,0 +1,13 @@
+package tests
+
+import (
+	"testing"
+)
+
+// TODO: Replace this test with real tests when glide client implementation is started
+func TestArbitraryLogic(t *testing.T) {
```

Comment:
I also added this arbitrary test so that the unit test workflow had something to run against